ga4gh/client.py

```diff
@@ -29,7 +54,89 @@ def __init__(self, urlPrefix, debugLevel, workarounds, key):
         self._workarounds = workarounds
         self._key = key
 
-    def runRequest(self, request, url, protocolClass, listAttr):
+    # TODO temporary auth solution
+    def _addAuth(self):
```

Comment:
_getAuth() would be more accurate here since we're returning something and not updating state.